src/transactionsFactories/tokenManagementTransactionsFactory.ts

```diff
@@ -24,10 +25,19 @@ interface IConfig {
     gasLimitEsdtNftAddQuantity: bigint;
     gasLimitEsdtNftBurn: bigint;
     gasLimitStorePerByte: bigint;
+    gasLimitEsdtModifyRoyalties: bigint;
+    gasLimitEsdtModifyCreator: bigint;
+    gasLimitEsdtMetadataUpdate: bigint;
+    gasLimitSetNewUris: bigint;
+    gasLimitNftMetadataRecreate: bigint;
+    gasLimitNftChangeToDynamic: bigint;
+    gasLimitUpdateTokenId: bigint;
+    gasLimitRegisterDynamic: bigint;
     issueCost: bigint;
 }
 
 type RegisterAndSetAllRolesTokenType = "NFT" | "SFT" | "META" | "FNG";
+type TokenTypes = "NFT" | "SFT" | "META" | "FNG";
```

Comment:
:+1:

Reply:
Maybe as singular, `TokenType`? Also, since `RegisterAndSetAllRolesTokenType` is not exported, maybe drop it and only keep the new one?